src/data/options.html

```diff
@@ -44,6 +44,9 @@ <h1 style="line-height:32px;height:32px;margin-top:16px;">Tumblr Savior <span id
           <div id="match_words_div">
             <input type="checkbox" id="match_words_cb" name="match_words_cb" /> <label for="match_words_cb">Match whole words only. Use * as a wildcard (ie. ip* would match ipad and iphone, but not flip or mudkips).</label>
           </div>
+          <div id="ignore_div">
+            Ignore matches in these fields: Blog name <input type="checkbox" id="ignore_header_cb" name="ignore_header_cb" /> Body <input type="checkbox" id="ignore_body_cb" name="ignore_body_cb" /> Tags <input type="checkbox" id="ignore_tags_cb" name="ignore_tags_cb" />
```

Comment:
Do you see how your new settings don't match the existing settings?

![screenshot from 2015-04-11 01-26-16](https://cloud.githubusercontent.com/assets/7987724/7100488/d745bfe6-dfe9-11e4-9d05-8ca6f578534d.png)

There is a  checkbox on the left and a label on the right.

Instead of `Blog name`, please use `Header`